### PR TITLE
[CUBRIDQA-55] add readme for ha_repl

### PR DESCRIPTION
**Pull request by supler**
@fanzaiqiang 

add ha_repl readme

### CODE REVIEW

File: CTP/README.md [hunk 1]
```diff
@@ -166,14 +166,38 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
...
+	  ```
+	     JAVA_HOME=$java_nstallation_directory 
+	     CTP_HOME=$HOME/CTP
+	  ```
+	  
+ - Run Tests 
+	* For **HA Replication** test:
+	
+	  ```
+	  $ bin/ctp.sh ha_repl -c ./conf/ha_repl_template.conf
```
Comment (fanzaiqiang) [inline CTP/README.md]: Rename conf/ha_repl_template.conf to ha_repl.conf

File: CTP/README.md [hunk 2]
```diff
@@ -321,6 +345,74 @@ You can find generated jar files ``common/lib/cubridqa-common.jar``, ``sql/lib/c
...
         - MC means main controller, it will control and coordinate the order of all transaction clients
 
+
+- **HA Replication**
+   * Test cases: Since ``HA Replication`` is using ``SQL`` scenarios to test on HA mode to verify the data synchronization between an active server and a standby server,
+     so the cases are same as ``SQL``
+   * CTP will transform case file to be ``case_name.test`` file with some checking statement flags around the SQL statement. And If the SQL does not contain primary key,
+     CTP will add primary key on column
+   * Sample for reference
+     ```
```
Comment (fanzaiqiang) [inline CTP/README.md]: Change to other typical test case.

Current test case has the feature of OO.

**Comment by supler** [comment]
@fanzaiqiang 

update it accordingly

**Comment by fanzaiqiang** [comment]
Great for all current changes.

We also need a file conf/ha_repl.conf.

**Comment by supler** [comment]
@fanzaiqiang 

add template conf for ha_repl

File: CTP/conf/ha_repl.conf [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+env.instance01.cubrid.cubrid_port_id=<cubrid port>
+env.instance01.ha.ha_port_id=<ha port>
+env.instance01.broker.BROKER_PORT=<broker port>
+
+# If execute continue testing when testing is interrupted
+main.mode.continue=false
+# The execution way on master, it supports jdbc and csql
+main.testmode=jdbc
+# The root path of test case, it should be checked out on controller node
+main.testcase.root=<path to test cases in remote test node>
```
Comment (fanzaiqiang) [inline CTP/conf/ha_repl.conf]: As is:
+main.testcase.root=<path to test cases in remote test node>

To be:
+main.testcase.root=<path to test cases in only controller node>

**Comment by fanzaiqiang** [comment]
@supler 

Great. Please merge all codes. Thanks.

**Pull request merged**